language/move-lang/src/errors/diagnostic_codes.rs

```diff
@@ -140,6 +141,7 @@ codes!(
     // errors for any unused code or items
     UnusedItem: [
         Alias: { msg: "unused alias", severity: Warning },
+        TypeParam: { msg: "unused type parameter", severity: Warning },
```

Comment:
About this, I originally intended to put this in the same PR, but I think that would require updating diem-framework to not warn on unused type parameters, no? Maybe best to leave it for a different PR to avoid bloating this one?

I implemented the check, but then remove it and forgot to remove this.

Reply:
Ah. Yeah let's save it for a future PR, after we migrate the DF and stdlib to `phantom`